Makefile

```diff
@@ -0,0 +1,28 @@
+.PHONY: fetch lint build local-image
+
+export CARGO_ENV_VARS = CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
```

Comment:
The `sparse`  protocol is the default now I think. We can probably omit setting this explicitly.